### DISCUSSION TITLE
盘古之白 - 中英文之间自动加空格 | /cn/2017/05/pangu/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#923] [Comments]
https://yihui.org/cn/2017/05/pangu/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
盘古和 xeCJK 的共同问题就是放在代码字体里的英文单词与代码之外的中文之间不会自动加空格，既然你也提出来了，我就去给盘古提交了一个报告：<https://github.com/vinta/pangu.js/issues/97>

我现在都是搜狗输入法自动加空格，从输入的层面就直接解决掉这个问题。

> ——原帖发布于 2017-05-17 18:24:42

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
哪里，我只是~~相声界~~前端码农界的一个小学生。CSS 基本靠瞄，JS 基本靠抄。随便写点 JS 都要在浏览器控制台中调试好久，永远记不住 .map / .forEach / .each 哪些是 jQuery 哪些是 JS 原生方法以及参数哪个是下标哪个是元素，所以通常只能用土鳖的 for 循环。《JavaScript: The Good Parts》看了前两章就昏死过去了。

> ——原帖发布于 2017-05-19 04:18:49

[a comment by an automated account is not shown]

**Comment by xfq** [comment]
浏览器现在可以自动加了，可以看看这篇文章：https://mp.weixin.qq.com/s/qj3wHJcn10aBIJZT3vQSYA

**Reply by yihui** [reply]
太感谢了！一行 CSS（ e45b2cb308cf6e4a10d599f436b1856d8296ca02 ）就解决了一个大问题，善哉善哉！
